new(tem): add a precision on TEM due to misunderstanding by clients (#1646)
scaleway/docs-content: managed-services/transactional-email/how-to/generate-api-keys-for-tem-with-iam.mdx

-d @mail.json
```
<Message type="important">

>  - If you have not configured your environment in your local machine, make sure you replace `\<project_id\>` with your [Project ID](console/my-project/concepts.mdx), `$REGION` with `fr-par` and `$TOKEN` with your secret key. If you don't replace the Project ID, the API will reply with a "permission denied" error message.

Reviewer: ```suggestion
 - If you have not configured your environment in your local machine, make sure you replace `\<project_id\>` with your [Project ID](/console/my-project/concepts/#project), `$REGION` with `fr-par` and `$TOKEN` with your secret key. If you do not replace the Project ID, the API displays a "permission denied" error message.
```
